frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -245,7 +249,7 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
   return (
     <div className="modal-content">
       <ModalTitle>
-        {isEditing ? 'Edit' : 'Add'} {type.toString()}
+        {isEditing ? EDIT : CREATE} {type.toString()}
```

Comment:
hmm, it stil makes sense to use this terminology in the wizard IMO. Because we are not creating them yet there. Could this be an optional prop (eg. submitButtonText)?

Reply:
Ah, it didn't occur to me that this is being used in the wizard, as well. An optional prop makes sense in that case. I think the modal title and submit button should match, though. So, I'll change the title based on the prop input as well, unless you see a problem with that.

Reply:
sure, sounds reasonable

Reply:
Done!